<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonChangePassword">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRegister">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLogin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_success">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录成功！</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUsername">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_failure">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录失败！该用户名不存在或输入密码错误！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonRegister</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>on_pushButtonRegister_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonChangePassword</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>on_pushButtonChangePassword_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButtonRegister_clicked()</slot>
  <slot>on_pushButtonChangePassword_clicked()</slot>
 </slots>
</ui>
